starter-core/src/test/groovy/io/micronaut/starter/feature/aws/AwsLambdaSnapstartSpec.groovy

```diff
@@ -0,0 +1,73 @@
+package io.micronaut.starter.feature.aws
+
+import io.micronaut.starter.BeanContextSpec
+import io.micronaut.starter.application.ApplicationType
+import io.micronaut.starter.feature.architecture.Arm
+import io.micronaut.starter.feature.architecture.CpuArchitecture
+import io.micronaut.starter.feature.architecture.X86
+import io.micronaut.starter.fixture.CommandOutputFixture
+import spock.lang.Shared
+import spock.lang.Subject
+import spock.lang.Unroll
+
+class AwsLambdaSnapstartSpec extends BeanContextSpec implements CommandOutputFixture {
+
+    @Shared
+    @Subject
+    AwsLambdaSnapstart snapstart = beanContext.getBean(AwsLambdaSnapstart)
+
+    void "is not visible"() {
+        expect:
+        !snapstart.isVisible()
+    }
+
+    @Unroll
```

Comment:
Quick headsup, `Unroll` is [the default now](https://github.com/spockframework/spock/issues/967#issuecomment-497016452)